src/glauth.py

```diff
@@ -32,7 +35,17 @@ def _snap():
     return cache["glauth"]
 
 
-def install():
+def create_default_config(api_port: int, ldap_port: int) -> None:
+    """Create default config with no users."""
+    with open("templates/glauth.toml.j2", "r") as f:
+        template = Template(f.read())
```

Comment:
Why not this instead?

```suggestion
    template = Template(pathlib.Path("templates/glauth.toml.j2").read_text())
```